cypress/integration/executions_spec.js

```diff
@@ -84,7 +84,7 @@ describe('Dashboard Executions Page', () => {
       );
 
       cy.contains('nav li a', 'Executions').as('executions');
-      cy.get('@executions').should('have.attr', 'href', '/executions');
+      cy.get('@executions').should('have.attr', 'href').and('include', '/executions');
```

Comment:
persisted `limit=x&page=y` parameters meant the href no longer only contains `/executions`.